docs/installation/windows.md

```diff
@@ -29,16 +29,19 @@ Once you've got this installed and after rebooting,
 On your first run, the system will ask for username and password. Take note of
 both since it will be used for `sudo` commands.
 
-### Ruby on WSL
+### Installing rbenv
 
-First, install Ruby language dependencies:
+`rbenv` is a version manager for Ruby applications which allows one to garuntee
```

Comment:
garuntee -> guarantee

I only noticed because I can't spell this word :laughing:

Reply:
I always misspell this word... 😆😆 should name the new commit `0 typos guaranteed!`...